src/ServiceFabric/ServiceFabric/Commands/UpdateAzServiceFabricApplication.cs

```diff
@@ -141,12 +141,12 @@ public class UpdateAzServiceFabricApplication : ProxyResourceCmdletBase
         [Parameter(Mandatory = false, ParameterSetName = ByResourceGroup)]
         [Parameter(Mandatory = false, ParameterSetName = ByResourceId)]
         [ValidateRange(0, 100)]
-        public int DefaultServiceTypeMaxPercentUnhealthyServices { get; set; }
+        public int DefaultServiceTypeUnhealthyServicesMaxPercent { get; set; }
```

Comment:
Some of the parameters missed a proper `HelpMessage`. It helps a lot for understanding complex concepts like this one so we should add it.

Reply:
yes, I had forgotten. done